edgelet/edgelet-core/Cargo.toml

```diff
@@ -6,22 +6,27 @@ publish = false
 edition = "2018"
 
 [dependencies]
+base64 = "0.9"
 bytes = "0.4"
 chrono = { version = "0.4", features = ["serde"] }
 consistenttime = "0.2.0"
 futures = "0.1"
 failure = "0.1"
 hmac = "0.5.0"
 lazy_static = "1.0"
+regex = "0.2"
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
 sha2 = "0.7.0"
 log = "0.4"
 url = "1.7"
+url_serde = "0.2"
 tokio = "0.1"
 
 edgelet-utils = { path = "../edgelet-utils" }
 
 [dev-dependencies]
 base64 = "0.9"
+
```

Comment:
not needed here?

Reply:
Are you talking about the new line? The convention has been to have one blank new line between the list of dependencies that are from crates.io and the list of dependencies that are local.

Reply:
You don't need the base64 dev dep since you've added it as a regular dep.

Why do you need the docker dev dep?

Reply:
Ah. Yes, `base64` is no longer needed here.

The docker dependency can be removed. It was added during one of the interim commits when I had the `DockerConfig` struct used in a test. Wish the "unused" warnings warned about unused dependency crates.

Reply:
Unfortunately unreferenced dependency crates are not necessarily unused since they can be used to contribute lang items. This is one of the casualties from edition 2018 removing `extern crate` statements.